app/scenes/settings/settings.js

```diff
@@ -129,6 +129,17 @@ export default class Settings extends PureComponent {
                             theme={theme}
                         />
                     }
+                    {config.HelpLink &&
+                        <SettingsItem
+                            defaultMessage='Help'
+                            i18nId='mobile.help.title'
+                            iconName='help'
+                            iconType='material'
+                            onPress={() => Linking.openURL(config.HelpLink)}
```

Comment:
can we surround this in a `preventDoubleTap` function?

like this

```javascript
preventDoubleTap(Linking.openURL, this, config.HelpLink)
```

thanks @cpanato